lib/bats-core/tracing.bash

```diff
@@ -23,6 +23,15 @@ bats_capture_stack_trace() {
 	done
 }
 
+bats_get_failure_stack_trace() {
+	# See bats_debug_trap for details.
+	if [[ -n "${BATS_DEBUG_LAST_STACK_TRACE_IS_VALID}" ]]; then
+		eval "${1}"'=("${BATS_DEBUG_LAST_STACK_TRACE[@]}")'
```

Comment:
I like combining `eval` with `printf` to make it easier to see what's being `eval`-ed. You could write this as:

    eval "$(printf '%s=("${BATS_DEBUG_LAST_STACK_TRACE[@]}")' "$1")"

Which makes it (more) obvious the only dynamic behavior is the variable name being assigned.